app/models/goal.py

```diff
@@ -3,4 +3,12 @@
 
 
 class Goal(db.Model):
-    goal_id = db.Column(db.Integer, primary_key=True)
+    goal_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
```

Comment:
Do we want to store goals with a `NULL` title? Consider adding `nullable=False` to ensure every goal requires a title.